package.json

```diff
@@ -512,7 +512,7 @@
     "glob": "7.1.3",
     "growl": "1.10.5",
     "he": "1.2.0",
-    "js-yaml": "3.13.0",
+    "js-yaml": "^3.13.1",
```

Comment:
Fix for this already merged

Reply:
Would it be possible to publish a new release since the vulnerability is high? Thanks